### PR TITLE
LPS-152382 Create condition function

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/test/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/ConditionFunctionTest.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+			new Object[][] {
+				{true, "expected", StringUtil.randomString(), "expected"},
+				{false, StringUtil.randomString(), "expected", "expected"},
+				{true, 2, 7, 2}, {false, 2, 7, 7},
+				{false, new BigDecimal(1), new BigDecimal(4), new BigDecimal(4)}
+			});
+	}
+
+	@Test
+	public void testConditionalFunction() {
```
Comment (MarinhoFeliphe) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/test/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/ConditionFunctionTest.java:57]: Rename to testApply

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:forward

**Comment by seltong** [comment]
ci:stop

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:stop

**Comment by seltong** [comment]
ci:forward

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:forward

**Comment by seltong** [comment]
ci:stop

**Comment by seltong** [comment]
reset here #140

**Pull request closed without merging**